lib/config/recommended.js

```diff
@@ -15,6 +15,9 @@ module.exports = {
     'style-concatenation': true,
     'deprecated-inline-view-helper': true,
     'simple-unless': true,
-    'unused-block-params': true
+    'unused-block-params': true,
+    'template-length': {
```

Comment:
In general, we don't update recommended ruleset when adding new rules. Adding new rules (or changing config) for recommended is often times a breaking change, and I don't think we are quite ready for another major version bump.  

Can you remove the changes to `lib/config/recommended.js`?

Reply:
Yes, makes sense. Will do!